uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseKeyboardOperation.java

```diff
@@ -98,16 +102,14 @@ private boolean isHeaderSelectionOriginSet(final GridWidget selectedGridWidget)
         return !gridModel.getSelectedHeaderCells().isEmpty();
     }
 
-    private boolean isGridWidgetRendered(final GridWidget gridWidget) {
-        final BaseGridRendererHelper rendererHelper = gridWidget.getRendererHelper();
-        final BaseGridRendererHelper.RenderingInformation renderingInformation = rendererHelper.getRenderingInformation();
+    private boolean isGridWidgetRendered(final BaseGridRendererHelper.RenderingInformation renderingInformation) {
```

Comment:
I'd be tempted to remove this method all together now.

Reply:
ok, that is questionable, it is one liner, but the method name gives a little bit more information, what the null check means